aplt/runner.py

```diff
@@ -291,8 +321,8 @@ def run_scenario(args=None, run=True):
     scenario = locate_function(arg)
     log.startLogging(sys.stdout)
     statsd_client = parse_statsd_args(arguments)
-    scenario_args = parse_string_to_list(arguments["SCENARIO_ARGS"] or "")
-    scenario_args = try_int_list_coerce(scenario_args)
+    scenario_args = try_int_list_coerce(arguments["SCENARIO_ARGS"])
```

Comment:
channeling you a bit, I'd argue that this probably should be arguments.get("SCENARIO_ARGS", ""), but i'm fine also noting that the breakage would be from the documentation being out of sync.